src/store/plugins/initSdk/index.js

```diff
@@ -236,6 +236,7 @@ export default (store) => {
       if (isEqual(currentNetwork, lastNetwork)) return;
       lastNetwork = currentNetwork;
       await recreateSdk();
+      store.state.sdk.selectNode(currentNetwork.name);
```

Comment:
Why don't we make it a part of `createSdk`? Also, is it true that sdk doesn't select node by default?

Reply:
It does, but sdk calls `selectNode` from `node-pool`, not from `wallet-rpc`, but notification about changing the node (which leads to `onNetworkChange` in aepp) comes only from wallet-rpc's `selectNode` (and it calls node-pool's `selectNode` btw)
https://github.com/aeternity/aepp-sdk-js/blob/23de7edfacc5830b987c868b4525d8de3860b6fa/es/utils/aepp-wallet-communication/rpc/wallet-rpc.js#L272-L280